.gitignore

```diff
@@ -61,3 +66,4 @@ props-table
 reports
 screenshots
 target
+dist
```

Comment:
can you remove dist in here, since we already have one in line 31?

Reply:
Removed: https://github.com/cerner/terra-graphs/pull/5/commits/3ccd122155cd4b49f219da9fd22211907e882ebb